### PR TITLE
Updates to matching API v1 changes

### PR DESCRIPTION
**Pull request by oudeismetis**
Connected to #38

### CODE REVIEW

**Comment by oudeismetis** [comment]
Not tested yet. I'm working to get my local running so I can test.
Ready for :eyes: though

File: api.go [hunk 1]
```diff
@@ -66,18 +59,16 @@ func init() {
...
 	if Config.ENV == "test" {
 		return
 	}
 
-	url := apiUrl(dockerInfo)
-
-	var jsonStr = []byte(fmt.Sprintf(`{"status":"%s"}`, data))
-	req, err := http.NewRequest("PUT", url, bytes.NewBuffer(jsonStr))
+  var jsonStr = []byte(fmt.Sprintf(`{"status":"%s","environment":"%s"}`, data, dockerInfo.ENVIRONMENT))
+	req, err := http.NewRequest("PUT", dockerInfo.CALLBACK, bytes.NewBuffer(jsonStr))
```
Comment (oudeismetis) [inline api.go]: odd formatting here. Not sure why this happened. Will check and fix.

**Comment by oudeismetis** [comment]
Everything seems to be good, but I'm getting an odd `http: multiple response.WriteHeader calls` error. The url and json payload for calling API back are correct. Not sure how to debug this further.

image

**Comment by JulianGindi** [comment]
This looks good. I'll test to see if I can reproduce the issue.

**Pull request merged**